frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -214,7 +214,8 @@ export const SidePanelSupport = (): JSX.Element => {
                                 </Section>
                             ) : null}
 
-                            {hasAvailableFeature(AvailableFeature.EMAIL_SUPPORT) ? (
+                            {hasAvailableFeature(AvailableFeature.EMAIL_SUPPORT) ||
+                            window.location.href.includes('/billing') ? (
```

Comment:
```suggestion
                            window.location.href.includes(urls.billing()) ? (
```

Want to use the `urls` object here instead of hard-coding the string. Prob will never change but it's a good habit to be in.

(I think I got the prop right but pull down locally to verify)